spaceship/lib/spaceship/connect_api/client.rb

```diff
@@ -274,12 +287,12 @@ def post_beta_app_review_submissions(build_id: nil)
           req.body = body.to_json
           req.headers['Content-Type'] = 'application/json'
         end
-        handle_response(response)
+        handle_response(response, only_data: only_data)
       end
 
       protected
 
-      def handle_response(response)
+      def handle_response(response, only_data: true)
```

Comment:
I would create a new method, maybe `handle_data_response`, which uses `handle_data` underneath. Let's try not to change any of the existing method signatures such as `post_beta_build_localizations`